Examples/BinaryBH/BinaryBHLevel.cpp

```diff
@@ -142,11 +142,11 @@ void BinaryBHLevel::specificPostTimeStep()
     {
         // Populate the Weyl Scalar values on the grid
         fillAllGhosts();
-        BoxLoops::loop(Weyl4(m_p.extraction_params.extraction_center, m_dx),
-                       m_state_new, m_state_new, EXCLUDE_GHOST_CELLS);
+        BoxLoops::loop(Weyl4(m_p.extraction_params.center, m_dx), m_state_new,
+                       m_state_new, EXCLUDE_GHOST_CELLS);
 
         // Do the extraction on the min extraction level
-        if (m_level == m_p.extraction_params.min_extraction_level)
+        if (m_level == m_p.extraction_params.min_extraction_level())
         {
             CH_TIME("WeylExtraction");
             // Now refresh the interpolator and do the interpolation
```

Comment:
Is it a good time to remove the `m_gr_amr.m_interpolator->refresh();` from the GRAMRLevel? We should move it to inside the extraction... It  would be a good opportunity now. The only problem is that old codes would do the refresh twice. But...

Reply:
No, I think this is a bad idea as the `refresh()` is quite expensive due to the exchange of ghost cells and if one is doing multiple separate extractions consecutively (e.g. Weyl and ADM mass), one only wants to do this once.

Reply:
True. So is this something that should be removed from inside the AHFinder? As one may want to extract Weyl and find the AH, so it is also being done twice in that case

Reply:
Probably but to be honest the AHFinder itself is going to be the main bottleneck there anyway.